packages/react/src/components/ComboBox/ComboBox.js

```diff
@@ -332,7 +336,7 @@ export default class ComboBox extends React.Component {
       placeholder,
       initialSelectedItem,
       selectedItem,
-      ariaLabel,
+      ariaLabel, // eslint-disable-line no-unused-vars
```

Comment:
I would imagine we should still be passing along the `ariaLabel` prop to downshift, right? @andreancardona

Reply:
Yes that was deleted on accident 🤦🏽‍♀️